lib/admin/templates/gallery/edit/layout.mustache

```diff
@@ -55,7 +55,24 @@
         }).mouseout(function(){
           $(this).find('.icon-ok').stop(true, true).fadeOut(200);
         });
-
+        
+        // Signup form actions
+        $('#signup-form form').submit(function(event){
+          event.preventDefault();
+          var form = $(this);
+          $.ajax({
+            method: form.attr('type'),
+            url: form.attr('action'),
+            data: form.serialize(),
+            dataType: 'json',
+          }).done(function(data){
+            $('#signup-form').empty().append("<span class='success'>Alright champ, you're signed up. check your inbox for your code.</span>");
```

Comment:
Can we add some sort of flag when the user subscribes to hide this CTA? No sense in showing it to them if they signed up.